src/@primer/gatsby-theme-doctocat/nav.yml

```
@@ -8,6 +8,17 @@
       url: /guides/zen
     - title: Figma
       url: /guides/figma
+    - title: Component Lifecycle
```

Nitpick: We should use sentence case here

```suggestion
    - title: Component lifecycle
```